scripts/check-remote-branch.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -euo pipefail
+
+function fail { >&2 echo -e "ERROR: $1"; exit 1; }
+
+(( $# == 1 )) || fail "Wrong number of arguments\nUsage: $0 <translation repo branch"
+BRANCH="$1"
+
+# check if sync branch exists
+if git ls-remote --exit-code --heads origin "$BRANCH"
+then
+    branch_exists=true
+    echo "Branch ${BRANCH} already exists"
+else
+    branch_exists=false
+    echo "Branch ${BRANCH} does not exist"
+fi
+
+echo "::set-output name=branch_exists::$branch_exists"
```

Comment:
Maybe it would be better to just return `$branch_exists` and let the github actions output to be defined in the workflow file? Otherwise, it is not very clear where the step output `branch_exists` came from.

Reply:
And then you would need to redirect the info messages to stderr as well, i.e.:
```sh
then
    branch_exists=true
    echo "Branch ${BRANCH} already exists" >&2
else
    branch_exists=false
    echo "Branch ${BRANCH} does not exist" >&2
fi
echo "$branch_exists"
```

Reply:
After looking at the other PR I understood why the outputs are set in the script instead of in the workflow. Please see here https://github.com/solidity-docs/.github/pull/41#discussion_r1051995063. So you can ignore this change.